src/_P077_CSE7766.ino

```diff
@@ -171,7 +172,7 @@ boolean Plugin_077(byte function, struct EventStruct *event, String& string)
 
             if (cse_receive_flag) {
               serial_in_buffer[serial_in_byte_counter++] = serial_in_byte;
-              if (24 == serial_in_byte_counter) {
+              if (24 >= serial_in_byte_counter) {
```

Comment:
I think this is the most important part of the fix

Reply:
Yes, now it is also buffer overlapping protection